test/check-ssl-hsts-preloadable_spec.rb

```diff
@@ -19,11 +19,11 @@
     expect { check.run }.to raise_error SystemExit
   end
 
-  it 'should pass check if the domain is preloadedable but has warnings' do
-    check.config[:domain] = 'oskuro.net'
-    expect(check).to receive(:warning).and_raise SystemExit
-    expect { check.run }.to raise_error SystemExit
-  end
+  #  it 'should pass check if the domain is preloadedable but has warnings' do
```

Comment:
Should these be uncommented or is there not a suitable domain to test against? I can't say I understand what's necessary but my digging has been surface. Happy to explore.